Net SDK Unit Tests/Integration Tests/EngineModuleTests.cs

```diff
@@ -1,214 +0,0 @@
-﻿using System;
```

Comment:
Yes, because Engine is gone too. I decided to remove it since GetTreatment responsability was divided in Client and Engine, so I prefered to merge the full GetTreatment funtionality to the same class, and it was easier to implement the different "label" cases when sending an impression.